exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Constraints.kt

```diff
@@ -51,30 +51,45 @@ enum class ReferenceOption {
  * Represents a foreign key constraint.
  */
 data class ForeignKeyConstraint(
-    val target: Column<*>,
-    val from: Column<*>,
+    val references: Map<Column<*>, Column<*>>,
     private val onUpdate: ReferenceOption?,
     private val onDelete: ReferenceOption?,
     private val name: String?
 ) : DdlAware {
+    constructor(
+        target: Column<*>,
+        from: Column<*>,
+        onUpdate: ReferenceOption?,
+        onDelete: ReferenceOption?,
+        name: String?
+    ) : this(mapOf(from to target), onUpdate, onDelete, name)
+
```

Comment:
I guess it's worth to add check that there are only unique target columns as Map allows duplicated values